components/Search/InkeepSearch.tsx

```diff
@@ -53,104 +74,87 @@ export function InkeepSearch() {
           type="text"
           className={styles["search-input"]}
           onChange={(e) => handleChange(e.target.value)}
-          onFocus={() => setIsOpen(true)}
+          onClick={() => setIsOpen(true)}
           placeholder="Search Docs"
           value={message}
         />
       </div>
-      <InkeepWidget
-        isOpen={isOpen}
-        onClose={() => setIsOpen(false)}
-        handleChange={handleChange}
-        chatCallableFunctionsRef={chatCallableFunctionsRef}
-        searchCallableFunctionsRef={searchCallableFunctionsRef}
-      />
+      <InkeepCustomTrigger {...inkeepCustomTriggerProps} />
     </div>
   );
 }
 
-function InkeepWidget({
-  chatCallableFunctionsRef,
-  searchCallableFunctionsRef,
-  handleChange,
-  isOpen,
-  onClose,
-}: {
-  chatCallableFunctionsRef: React.MutableRefObject<AIChatCallableFunctions>;
-  searchCallableFunctionsRef: React.MutableRefObject<SearchCallableFunctions>;
-  handleChange: (str: string) => void;
-  isOpen: boolean;
-  onClose: () => void;
-}) {
-  const version = useContext(DocsContext).versions.current;
-
-  const inkeepBaseSettings: InkeepWidgetBaseProps = {
-    apiKey: API_KEY,
-    integrationId: INTEGRATION_ID,
-    organizationId: "teleport",
-    organizationDisplayName: "Teleport",
-    primaryBrandColor: "#512FC9",
-    productVersion: version,
-    product: "Teleport",
-    publicSearchResultSources: ["DOCUMENTATION", "GITHUB_ISSUE"],
-    inkeepMentionDecoratorText: "Powered by ",
-    searchAndChatServiceProxyDomain: "goteleport.com/inkeep-proxy",
-    remoteErrorLoggingLevel: 0,
-    isOptedOutAllAnalytics: true,
-    theme: {
-      zIndices: {
-        overlay: 2100,
-        modal: 2200,
-        popover: 2300,
-        skipLink: 2400,
-        toast: 2500,
-        tooltip: 2600,
+const inkeepBaseSettings: InkeepWidgetBaseSettings = {
+  apiKey: API_KEY,
+  integrationId: INTEGRATION_ID,
+  organizationId: "teleport",
+  organizationDisplayName: "Teleport",
+  primaryBrandColor: "#512FC9",
+  product: "Teleport",
+  isInkeepMentionEnabled: true,
+  apiProxyDomain: "goteleport.com/inkeep-proxy",
+  remoteErrorLogsLevel: RemoteErrorLogsLevel.AnonymousErrors,
+  optOutAllAnalytics: true,
```

Comment:
I have a feeling the current PostHog analytics are messing with ours, I'll investigate more once merged.